testsupport/nexus-test-common/pom.xml

```diff
@@ -69,8 +69,8 @@
     </dependency>
 
     <dependency>
-      <groupId>org.eclipse.jetty.orbit</groupId>
-      <artifactId>javax.servlet</artifactId>
+      <groupId>javax.servlet</groupId>
+      <artifactId>javax.servlet-api</artifactId>
```

Comment:
Because 3.1 isn't available on central: http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.eclipse.jetty.orbit%22%20AND%20a%3A%22javax.servlet%22

The official 3.1 servlet-api jar is already packaged as a bundle, and is used by the Jetty 9 maven builds: https://github.com/eclipse/jetty.project/commit/958d0c6949b7d3d188a6eef51318f099f0928cad